iron-collapse.html

```diff
@@ -124,6 +124,23 @@
       this._enableTransition = true;
     },
 
+    attached: function() {
+      this._contentObserver = Polymer.dom(this.$.content).observeNodes(
+        function onContentChanged(){
+          if (this.opened) {
+            // ensure we consider the whole height
+            this.updateSize('auto', false);
+            this.updateSize(this._calcSize(), false);
```

Comment:
this is probably not needed, since it should be enough to set the size to 'auto'